src/main/java/uk/gov/hmcts/reform/hmc/config/MessageReaderFromTopicConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.hmc.config;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusFailureReason;
+import com.azure.messaging.servicebus.ServiceBusProcessorClient;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.hmc.ApplicationParams;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Component
+public class MessageReaderFromTopicConfiguration {
+
+    private final ApplicationParams applicationParams;
+
+    public MessageReaderFromTopicConfiguration(ApplicationParams applicationParams) {
+        this.applicationParams = applicationParams;
+    }
+
+    @Async
+    @EventListener(ApplicationStartedEvent.class)
+    @SuppressWarnings("squid:S2189")
+    public void readMessageFromTopic() {
+        CountDownLatch countdownLatch = new CountDownLatch(1);
+
+        // Create an instance of the processor through the ServiceBusClientBuilder
+        ServiceBusProcessorClient processorClient = new ServiceBusClientBuilder()
+            .connectionString(applicationParams.getConnectionString())
+            .processor()
+            .topicName(applicationParams.getTopicName())
+            .subscriptionName(applicationParams.getSubscriptionName())
+            .processMessage(MessageReaderFromTopicConfiguration::processMessage)
```

Comment:
```suggestion
            .processMessage(this::processMessage)
```

Sorry didn't notice on first run-through that `processMessage()` and `processError()` are both marked as `static`. Can we remove the `static` from them please - updating this line should allow that.